pointcloud/basic_octree_test.go

```diff
@@ -666,7 +666,7 @@ func TestCachedMaxProbability(t *testing.T) {
 		filledNode := basicOctreeNode{
 			children: nil,
 			nodeType: leafNodeFilled,
-			point:    PointAndData{P: r3.Vector{}, D: d},
+			point:    &PointAndData{P: r3.Vector{}, D: d},
```

Comment:
What is this test's purpose? A filledNode with no point (r3.Vector{}) is impossible to be set though nominal operation

Reply:
`r3.Vector{}` means `r3.Vector{0,0,0}`

Reply:
Understood. I personally would add in the 0,0,0 there for clarity and ease of understanding but you can leave it as is if you prefer this representation

Reply:
Good point. I will be explicit.

Technically this test is valid for any point as long as it's properly set.